spec/features/proposal/collection_draft_proposals/collection_draft_proposal_list_spec.rb

```diff
@@ -1,47 +1,186 @@
-describe 'Viewing Unsubmitted Collection Draft Proposals', js: true do
-  before do
-    login
-    set_as_proposal_mode_mmt(with_required_acl: true)
-    4.times { create(:full_collection_draft_proposal) }
-    create(:full_collection_draft_proposal, draft_short_name: 'An Example Proposal', version: '5', draft_entry_title: 'An Example Title')
-    create(:full_collection_draft_proposal, draft_short_name: 'A Second Example Proposal', version: '')
-  end
-
-  context 'while on the manage collection proposals page' do
+describe 'Viewing Collection Draft Proposals Index Pages', js: true do
```

Comment:
This file diffs poorly because there was a formatting change when I moved some of the existing tests into a new wrapper context block for 'user' view.  I will try to highlight real changes.